src/components/Button/Button.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+import './Button.css';
+
+/**
+ * A Simple rounded button
+ */
+const Button = ({ size, color, outline, disabled, children }) => {
+
+  const buttonClass = className(
+    'Button',
+    {[`Button-${size}`]: ['small', 'large'].includes(size)},
+    `Button${outline ? '-outline' : ''}-${color}`,
+  );
+  return (
+    <button type='button' className={buttonClass} disabled={disabled}>
```

Comment:
is the `type` prop for accessibility?

Reply:
Buttons are `submit` by default, which usually isn't what's wanted.